reactnative/RNPlayground/app/models/PlaygroundStore.ts

```diff
@@ -0,0 +1,79 @@
+import { types } from "mobx-state-tree";
+import { delay } from "../utils/delay";
+
+const Content = types.model("Content", { number: 0 })
+const Loading = types.model("Loading")
+const Error = types.model("Error", { message: "" })
+const PlaygroundUiState = types
+    .model('PlaygroundUiState', {
+        loading: types.maybe(Loading),
+        error: types.maybe(Error),
+        content: types.maybe(Content)
```

Comment:
When would we use `types.maybe()` vs. `types.optional()`? (`line 41`)

Reply:
If I understand correctly optional means that if the value is not there (i.e. persisted from last session) then it would set the default value provided as the second argument. Maybe seems like the actual nullable type (with two variants, one uses `null` and the other `undefined` to express missing value)